packages/react/src/themes/teams/components/Tree/treeTitleVariables.ts

```diff
@@ -0,0 +1,9 @@
+export interface TreeTitleVariables {
+  defaultColor: string
+}
+
+export default (siteVars: any): TreeTitleVariables => {
+  return {
+    defaultColor: siteVars.colors.grey[750],
```

Comment:
https://stardust-ui.github.io/react/color-schemes
This color perfectly maps `foreground` from `default`, can we use it? And we will be to remove variable overrides in Dark and HC themes.

I also suggest to rename it to `color` to match our variable naming convention:

```suggestion
    color: siteVars.colorScheme.default.foreground,
```